server/routes.go

```diff
@@ -70,7 +70,7 @@ func routesCreateHandler(writer http.ResponseWriter, request *http.Request) {
 		return
 	}
 
-	Routes.CreateMapping(definition.ServerAddress, definition.Backend, func(ctx context.Context) error { return nil })
+	Routes.CreateMapping(definition.ServerAddress, definition.Backend, func(ctx context.Context) error { return nil }, func(ctx context.Context) error { return nil })
```

Comment:
It's probably time to declare a typedef for a `ScalerFunc` that is `func(ctx context.Context) error`. These growing return declarations will likely be easier to interpret.